CRM/Contact/Form/Search/Criteria.php

```diff
@@ -540,9 +540,6 @@ public static function relationship(&$form) {
     }
     CRM_Core_Form_Date::buildDateRange($form, 'relation_active_period_date', 1, '_low', '_high', ts('From:'), FALSE, FALSE);
 
-    // Add reltionship dates
```

Comment:
I couldn't see this field exposed at all on the form so lets bin it

Reply:
@eileenmcnaughton looks like this is when the relation_date field was stripped off the form https://github.com/civicrm/civicrm-core/commit/ad6821345fe5911f9d1ce02ae2e7b9627d8651ec#diff-0ac8fd8c66c6d1e69b6dafa540ecddbf